### PR TITLE
Update to latest PHP, Python, and Composer

### PR DESCRIPTION
**Pull request by davereid**

### CODE REVIEW

**Comment by davereid** [comment]
Unsure of how to update the README.md that references all this.

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # elecena.pl (c) 2015-2023
 
 # https://hub.docker.com/_/php
-ARG PHP_VERSION=8.2.3
+ARG PHP_VERSION=8.2.12
```
Comment (macbre) [inline Dockerfile:4]:
```suggestion
ARG PHP_VERSION=8.2.16
```

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 # https://hub.docker.com/_/python/
-ARG PYTHON_VERSION=3.11.2
+ARG PYTHON_VERSION=3.12.0
```
Comment (macbre) [inline Dockerfile:7]:
```suggestion
ARG PYTHON_VERSION=3.12.2
```

File: Dockerfile [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
 # https://hub.docker.com/_/composer
-ARG COMPOSER_VERSION=2.5.4
+ARG COMPOSER_VERSION=2.6.5
```
Comment (macbre) [inline Dockerfile:10]:
```suggestion
ARG COMPOSER_VERSION=2.7.1
```

**Comment by macbre** [comment]
> Unsure of how to update the README.md that references all this.

No worries, I'll take of that. Thanks for the PR!

**Pull request merged**